libraries/AP_Baro/AP_Baro.h

```diff
@@ -177,6 +180,8 @@ class AP_Baro
         float altitude;                 // calculated altitude
         AP_Float ground_pressure;
         float p_correction;
+        uint8_t can_manager;
+        uint8_t uavcan_node;
```

Comment:
Because UAVCAN library can be instantiated several times and they do not know about each other. It is property of baro sensor to know where it is connected and what node number it has.